src/modules/MessageSearch/components/MessageSearchUI/index.tsx

```diff
@@ -108,7 +98,7 @@ export const MessageSearchUI: React.FC<MessageSearchUIProps> = ({
       <div className="sendbird-message-search">
         <PlaceHolder
           type={PlaceHolderTypes.SEARCH_IN}
-          searchInString={getChannelName()}
+          searchInString={getChannelTitle(currentChannel, '', stringSet)}
```

Comment:
I wasn't sure if it'd be safe to add this hook to this component:
```ts
  const state = useSendbirdStateContext();
  const userId = state?.config?.userId;
```

(passing `userId = ''` should be safe if that hook is no good... it'd mean no behavior change here - the prior `getChannelName` function was already not incorporating currentUserId when joining members)